### DISCUSSION TITLE
Coding suggestions

### DISCUSSION THREAD

**Discussion by Mister-Hope** [ArtalkJS/Artalk#685] [General]
It's not a good habit to renamed default export.

E.g.: 

- https://github.com/ArtalkJS/Artalk/blob/763fbc417ea283ec442a88ac50c46c3a66ca108d/ui/packages/artalk/src/artalk.ts#L5 

   There is actually no need to rename Context class

- https://github.com/ArtalkJS/Artalk/blob/763fbc417ea283ec442a88ac50c46c3a66ca108d/ui/packages/artalk/src/lib/user.ts#L55

   While all imports are `import User from './user'` and the file has a `User` class

Another case is:

- https://github.com/ArtalkJS/Artalk/blob/763fbc417ea283ec442a88ac50c46c3a66ca108d/ui/packages/artalk/src/plugins/markdown.ts#L1-L10

 
 
 https://github.com/ArtalkJS/Artalk/blob/763fbc417ea283ec442a88ac50c46c3a66ca108d/ui/packages/artalk/src/comment/comment.ts#L171

   Marked is a standard library, while the same variable is used on a local edit of the standard marked.  

   Contributors will be confused about API like this and likely to make mistakes by writing library api when contributing.

The above things will greatly confuse others when understanding symbols in the project.

To solve the first one, import the default export with the exact name, for the second one, you'd better use names like `myMarked` or `markdownParser`

**Comment by qwqcode** [comment]
I got it! It is very helpful! I will improve one by one. ❤️
